skill-tree /src/main/java/com/RDS/skilltree/enums/Behaviour.java

```diff
@@ -0,0 +1,6 @@
+package com.RDS.skilltree.enums;
+
+public enum Behaviour {
```

Comment:
can't we have common enum for Behaviour and EndorsementType

Reply:
Yes we can, will make this change